documentation/docs/practical-sessions/querying-data-using-the-wis2box-api.md

```diff
@@ -37,11 +46,17 @@ Click on the 'Browse' link, then click on the 'json' link.
 !!! question
     How many stations are returned? Compare this number to the station list in `/data/wis2box/metadata/station/station_list.csv` file when logged into wis2box (`python3 wis2box-ctl.py login`).
 
+??? success "Click to reveal answer"
```

Comment:
"Click to reveal answer" or "Click to reveal the answer"? I think we've used both and this is only a minor nitpick.